drizzlepac/hapsequencer.py

```diff
@@ -334,6 +335,23 @@ def run_hla_processing(input_filename, result=None, debug=False):
         # instrument_programID_obsetID_manifest.txt (e.g.,wfc3_b46_06_manifest.txt)
         manifest_name = total_list[0].manifest_name
 
+        # Set up all pipeline parameter values that will be used later on in the run.
+        # First, build temp.list of all product objects
+        full_product_list = total_list.copy()
+        for total_item in total_list:
+            full_product_list += total_item.fdp_list
+            full_product_list += total_item.edp_list
+
+        # Next, generate output parameter file
+        param_out_filename = input_filename.replace(".out", "_config.json")
+
+        # Finally, instantiate configuration object in each product object
+        for item in full_product_list:
+            item.pars = config_utils.HapConfig(item, output_custom_pars_file=param_out_filename, use_defaults=True)
+
+        # Lastly, delete temp.list of all product objects 
+        del full_product_list
```

Comment:
To go along with the previous comment about line 340, we do not want to work on a copy of the product list, so there will be nothing to delete, so this line can be removed.

Reply:
Resolved by commit [33fc25c](https://github.com/spacetelescope/drizzlepac/pull/439/commits/33fc25c59307379fe90008fea1e7de121c4f0c93)